pkg/manager/member/tikv_upgrader.go

```diff
@@ -43,6 +44,7 @@ const (
 	// TODO: change to use minReadySeconds in sts spec
 	// See https://kubernetes.io/blog/2021/08/27/minreadyseconds-statefulsets/
 	annoKeyTiKVMinReadySeconds = "tidb.pingcap.com/tikv-min-ready-seconds"
+	annoKeySkipStoreStateCheck = "tidb.pingcap.com/skip-store-check-for-upgrade"
```

Comment:
How about changing from `skip: false` to `check: true`?

In other words, renaming the anno key to something like `tidb.pingcap.com/store-check-for-upgrade`, and only take effect if the value is `true`.

BTW, this is only for the TiKV store and not for the TiFlash store, so how about renaming it to include `tikv` in the key name or adding some comments to explain it?